etcd-perf/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM centos:7
```

Comment:
Curious if we could use centos 8?

Reply:
Updated the base image to centos 8. Thanks.